PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterFrom.java

```diff
@@ -63,18 +66,23 @@
     private JTextField userNameField;
     private JPasswordField passwordField;
     private JTextField errorMessageField;
-    private JPanel buttonPanel;
-    private JButton Okbutton;
-    private JButton cancelButton;
     private JTextField storageNameField;
-    private JTextField storageKeyTextField;
+    private JTextArea storageKeyTextField;
+    private JLabel clusterNameLabel;
+    private JLabel storageNameLabel;
+    private JLabel storageKeyLabel;
+    private JLabel userNameLabel;
+    private JLabel passwordLabel;
 
     private HDInsightRootModule hdInsightModule;
 
     private static final String URL_PREFIX = "https://";
+    private static final String HELP_URL = "https://go.microsoft.com/fwlink/?linkid=866472";
 
     public AddNewClusterFrom(final Project project, HDInsightRootModule hdInsightModule) {
         super(project, true);
+        myHelpAction = new HelpAction();
```

Comment:
>myHelpAction = new HelpAction(); [](start = 8, length = 32)

1. Can we add a "HyperLink" in the dialogue so users can click/know if they run into issues?
2. DialogWrapper seems already have HelpAction defined, no need to add that part of code here?

Reply:
Or add tooltip

---
In reply to: [161944649](https://github.com/Microsoft/azure-tools-for-java/pull/1340#discussion_r161944649) [](ancestors = 161944649)

Reply:
Currently, the Intellij style 'help' icon is help to open helplink. The HelpAction defined by DialogWrapper is only used for Intellij internal help DB with ID reference. Fix it by adding more details into tooltip.